controllers/provider-aws/example/kube-apiserver.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
```

Comment:
Isn't https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-aws/example/10-fake-shoot-controlplane.yaml#L84-L99 good enough already?

Reply:
Sorry, I didn't need to upload this file. I will remove it.